src/styles/color_theme.scss

```diff
@@ -629,3 +571,145 @@ select {
     border: 1px solid var(--br-primary);
     color: var(--text-panel);
 }
+
+.smallscreen-navbar {
+    background-color: var(--primary);
+    color: var(--text-panel);
+}
+.smallscreen-navbar-inner li {
+    background-color: var(--text-panel);
+    border: 1px solid var(--br-secondary);
+    color: var(--primary);
+}
+.SmallScreen-Navbar-li {
+    background-color: var(--text-panel);
+    color: var(--primary);
+}
+#smallNavbarMenu-btn {
+    background-color: var(--primary);
+    color: var(--text-panel);
+}
+.touch-Ce-Menu {
+    background-color: var(--primary);
+    border: 1px solid var(--br-secondary);
+    color: var(--text-panel);
+}
+#smallNavbarMenu-btn::before {
+    background: var(--text-panel);
+    box-shadow: 0 5px 0 var(--text-panel);
+}
+#smallNavbarMenu-btn::after {
+    background: var(--text-panel);
+}
+#smallNavbarMenu-btn.active::before {
+    box-shadow: 0 0 0 var(--text-panel);
+}
+
+#TouchCe-panel::before {
+    border-color: transparent transparent var(--br-secondary);
+}
+#touch-module-property {
+    background-color: var(--primary);
+    border: 1px solid var(--br-secondary);
+    color: var(--text-panel);
+}
+.mobilepanelHeader:after,
+.mobilepanelHeader:before {
+    background-color: var(--br-primary);
+    border: 1px solid var(--br-primary);
+}
+#touchElement-property:before {
+    border-color: transparent transparent var(--br-secondary) transparent;
+}
+#touchtD-popover {
+    background-color: var(--primary);
+    border: 1px solid var(--br-secondary);
+    color: var(--text-panel);
+}
+#touchtD-popover::before {
+    border-color: transparent transparent var(--br-secondary) transparent;
+}
+#touch-time-daigram {
+    background-color: var(--primary);
+    border: 1px solid var(--br-secondary);
+    color: var(--text-panel);
+}
+
+#touch-timing-diagram-toolbar {
+    background-color: var(--bg-tabs);
+}
+#quickMenu {
+    background-color: var(--primary);
+    border: 1px solid var(--br-secondary);
+    color: var(--text-panel);
+}
+#quickmenu-Popover {
+    background-color: var(--primary);
+    border: 1px solid var(--br-secondary);
+    color: var(--text-panel);
+}
+#touchElement-property:before {
+    border-color: transparent transparent var(--br-secondary) transparent;
+}
+#quickmenu-Popover::before {
+    border-color: transparent transparent var(--br-secondary);
+}
+.quicMenu-align {
+    background-color: var(--text-panel);
+    color: var(--primary);
+}
+.touchMenuIcon {
+    background-color: var(--primary);
+    border: 2px solid var(--br-secondary);
+    color: var(--text-panel);
+}
+<<<<<<< HEAD
+=======
```

Comment:
@DhairyaMajmudar Could you fix the git conflict issues ?

Reply:
Yes I am removing conflicts and removing unwanted commits

Reply:
@tachyons I have resolved the merge conflicts and also reverted unwanted changes pls . review pr and merge it.